src/platform/packages/shared/kbn-unified-tabs/src/components/tabbed_content/tabbed_content.tsx

```diff
@@ -143,7 +143,24 @@ export const TabbedContent: React.FC<TabbedContentProps> = ({
       maxItemsCount,
       onDuplicate: (item) => {
         const newItem = createItem();
-        newItem.label = `${item.label} (copy)`;
+        const baseLabel = item.label.includes('(copy)')
+          ? item.label.replace(/\s*\(copy\)( \d+)?$/, '')
+          : item.label;
```

Comment:
```suggestion
        const baseLabel = item.label.replace(/\s*\(copy\)( \d+)?$/, '');
```
Nit: you might not need the condition here.